modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -56,6 +56,7 @@ Resource intensive applications might require additional CPU capacity. If an app
 * The CPU limit is upper-bounded by the CPU cores provided on the worker nodes.
 * The maximum recommended utilization per CPU core is 20 - 25 simple applications and API gateways. More complex applications with higher usage may need more capacity.
 * Runtime Fabric on VMs / Bare Metal runs a small amount of services on worker nodes. These services consume between 0.3 and 0.5 vCPU, depending on whether log forwarding is enabled for Anypoint Monitoring.
+* Each application replica must reserve at least 50m of CPU and 50 Mi of memory for the `anypoint-monitoring` container. This container is used to stream application metrics to the control plane and cannot be disabled.
```

Comment:
This currently lands as a bullet point for "When using CPU bursting, you must consider the following factors:" -- when it should be considered regardless of whether CPU bursting is used or not. Consider moving so it's considered for any deployment.

Consider removing "application" when mentioning "application replica" as this would apply to API gateways as well.

The 50m of CPU isn't well understood for our audience, so we should normalize to 0.05 CPU cores. We should also make it clear these CPU cores are not used for Mule licensing.

We mention "anypoint-monitoring container", which is more technical than we need to be. Instead, we could refer to this as the Anypoint Monitoring agent.

Finally, this is a consideration we want the customer to make when they size their RTF worker nodes -- not when they deploy their application or gateway (as they don't have control over the amount of CPU and memory given to the Anypoint Monitoring agent). We could phrase it accordingly -- as an example: 

"When estimating the CPU and memory required for your worker nodes, it's important to consider the Anypoint Monitoring agent requires ...."